components/script/dom/htmlinputelement.rs

```diff
@@ -1019,7 +991,7 @@ impl HTMLInputElement {
     }
 
     // https://html.spec.whatwg.org/multipage/#value-sanitization-algorithm
-    fn sanitize_value(&self) {
+    fn sanitize_value(&self, update_text_cursor: bool ) {
```

Comment:
I don't think we need to pass an argument here. All of the callers will want the behaviour we get from passing `true`.

Reply:
But sanitize_value is the function that calls set_content. And sanitize_value is being called from update_text_contents also.